packages/generators/src/handlers/default.ts

```diff
@@ -56,7 +56,7 @@ export async function questions(self: CustomGenerator, Question: Record<string,
         self,
         'cssType',
         'Which of the following CSS solutions do you want to use?',
-        ['none', 'CSS', 'SASS', 'LESS', 'Stylus'],
+        ['none', 'CSS', 'SASS', 'LESS', 'Stylus', 'PostCSS'],
```

Comment:
I think we need refactor here, ideally:
- Do you want to use CSS? - None/Yes (style-loader and css-loader), will be great to add `mini-css-extract-plugin` when mode is production, but we can do it in separate PR
- Do you want to use `CSS` preprocessor: `SASS`/`Less`/`Stylus`
- Do you want to use `CSS` postprocessolor : `postcss`

What do you think?

Reply:
`mini-css-extract-plugin` seems like like a very good idea, will add this in.
Meanwhile, I couldn't understand how adding three separate questions for it seems to bring any value as it is always nice to keep things simple and quick to use. A little more explanation on your intentions would be nice.

Reply:
Some developers use `postcss`, some not, maybe we can do it in our questions. 

Do you need CSS?
- None
- CSS
- CSS and PostCSS
- CSS and SASS (for less and Stylus too)
- CSS, PostCSS and SASS (for less and Stylus too)

I was thinking about separate this into different questions... There are a lot of answers...

Reply:
Hmm, I understood. I think I would do that in seperate PR. Currently in this PR, I am adding PostCSS  with CSS only.

Reply:
Yep, let's do it in separate PR